src/transformer/descriptor/helper/helper.ts

```diff
@@ -16,7 +16,11 @@ export namespace TypescriptHelper {
 
   export function GetDeclarationFromNode(node: ts.Node): ts.Declaration {
     const typeChecker: ts.TypeChecker = TypeChecker();
-    const symbol: ts.Symbol = typeChecker.getSymbolAtLocation(node);
+    const symbol: ts.Symbol | undefined = typeChecker.getSymbolAtLocation(node);
```

Comment:
So, this could happen, true. I am not sure when. We didn't have any issues with this yet. (Cannot read name of undefined). I would either investigate when this could happen or I would put some more useful error message.